code/controllers/subsystem/movement/ai_movement.dm

```diff
@@ -1,20 +1,19 @@
 /// The subsystem used to tick [/datum/ai_movement] instances. Handling the movement of individual AI instances
-PROCESSING_SUBSYSTEM_DEF(ai_movement)
+MOVEMENT_SUBSYSTEM_DEF(ai_movement)
 	name = "AI movement"
-	flags = SS_KEEP_TIMING|SS_BACKGROUND
+	flags = SS_KEEP_TIMING|SS_TICKER
```

Comment:
Not having SS_BACKGROUND is more because I wasn't sure of the point of it, though I suppose that means I should just keep it.
Move looks need to be SS_TICKER because I do kinda want the behavior that provides. So A: cleanish tick based waits, and B: We do all our loop processing early, as if we were processing later. Saves some sanity. Though honestly maybe I don't want that. Huh. Lemme do this the hack way.

Reply:
Changed my mind. See commit
https://github.com/tgstation/tgstation/pull/62567/commits/f6ddeeafb915c1e67fe2c14044a75579f6117209